docs/docs/examples/mysql.md

```diff
@@ -0,0 +1,251 @@
+`/contrast/secrets/workload-secret-seed` to generate a key and setup the block
+device as a LUKS partition. Before starting the MySQL container, the init
+container uses the generated key to open the LUKS device, which is then mounted
+by the MySQL container. For the MySQL container, this process is completely
+transparent and works like mounting any other volume.
+
+## Verifying the deployment as a user
+
+In different scenarios, users of an app may want to verify its security and identity before sharing data, for example, before connecting to the database.
+With Contrast, a user only needs a single remote-attestation step to verify the deployment - regardless of the size or scale of the deployment.
+Contrast is designed such that, by verifying the Coordinator, the user transitively verifies those systems the Coordinator has already verified or will verify in the future.
+Successful verification of the Coordinator means that the user can be sure that the given manifest will be enforced.
+
+### Verifying the Coordinator
+
+A user can verify the Contrast deployment using the verify
+command:
+
+```sh
+contrast verify -c "${coordinator}:1313" -m manifest.json
+```
+
+The CLI will verify the Coordinator via remote attestation using the reference values from a given manifest. This manifest needs
+to be communicated out of band to everyone wanting to verify the deployment, as the `verify` command checks
+if the currently active manifest at the Coordinator matches the manifest given to the CLI. If the command succeeds,
+the Coordinator deployment was successfully verified to be running in the expected Confidential
+Computing environment with the expected code version. The Coordinator will then return its
+configuration over the established TLS channel. The CLI will store this information, namely the root
+certificate of the mesh (`mesh-ca.pem`) and the history of manifests, into the `verify/` directory.
+In addition, the policies referenced in the manifest history are also written into the same directory.
+
+:::warning
+On bare metal, the [coordinator policy hash](components/policies.md#platform-differences) must be overwritten using `--coordinator-policy-hash`.
+:::
+
+### Auditing the manifest history and artifacts
+
+In the next step, the Coordinator configuration that was written by the `verify` command needs to be audited.
+A user of the application should inspect the manifest and the referenced policies. They could delegate
+this task to an entity they trust.
+
+### Connecting to the application
+
+To access the MySQL database from the command line, expose the service on a public IP address via a LoadBalancer service:
+
+```sh
+mysqlIP=$(kubectl get svc mysql-svc -o=jsonpath='{.status.loadBalancer.ingress[0].ip}')
+echo "MySQL is available at the IP  $mysqlIP."
+```
+
+Using the `mysql` command line interface, you can connect to the database as the
+root user with the password specified in the deployment YAML (`password`).
```

Comment:
I think this setup does not sound ideal, even for a demo. Could we maybe mention the service mesh here to ensure only Contrast deployments can access the MySQL server?